### PR TITLE
Improved performance of presenting animation

### PR DESCRIPTION
**Pull request by rinat-gabdullin**
Hi!

The performance of iPhone 6 Plus isn't enough to present a view controller smoothly even in the example project. Because using a mask layer for a root view reduces it dramatically. There a solution that makes the presenting animation as smooth as supposed to be.

### CODE REVIEW

File: DeckTransition/Classes/RoundedView.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override func awakeFromNib() {
```
Comment (ebaker355) [inline DeckTransition/Classes/RoundedView.swift]: Should call super.awakeFromNib()

Comment (rinat-gabdullin) [inline DeckTransition/Classes/RoundedView.swift]: Thanks. Fixed.

**Comment by HarshilShah** [comment]
Hi Rinat, and thanks for the PR.

I am not comfortable at all messing with the view hierarchy for any view controllers that I don’t entirely control, so if there’s any way to implement this without touching the view controllers, that’d be great.

Additionally, iOS 11 includes new API to round corners. I’m not sure if they trigger offscreen rendering as well, but I’ll be taking a look into that and implementing it for iOS 11+

**Comment by HarshilShah** [comment]
This solution doesn't accommodate overscroll and also touches the external view hierarchy. I have an alternative in the works that takes care of both issues, so closing this

**Comment by HarshilShah** [comment]
Reopening this. Gonna be including the `RoundedView` in a future release

**Comment by rinat-gabdullin** [comment]
Hi Harshil. 

Perhaps it is possible to insert `RoundedView` into `transitionContext.containerView` instead of `presentedViewController.view`.

**Comment by HarshilShah** [comment]
Yup, that's what I'm doing now. If it isn't too much of a bother, would you mind updating your PR to be based on the current state of the repo? I restructured the entire repo in #23, so even sorting merge conflicts is a bit difficult

**Comment by HarshilShah** [comment]
Implemented in #25, and will be released as a part of version 1.1

**Pull request closed without merging**